From: Koray Gulcu <koray.gulcu@ozu.edu.tr>
Subject: [PATCH] Makefile: change EXTRAVERSION field to contain -eudyptula

As per eudyptula challenge task03, change EXTRAVERSION field to contain "-eudyptula"
Signed-off-by: Koray Gulcu <koray.gulcu@ozu.edu.tr>
---
 Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 041c685..b11db66 100644
--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,7 @@
 VERSION = 3
 PATCHLEVEL = 15
 SUBLEVEL = 0
-EXTRAVERSION = -rc3
+EXTRAVERSION = -eudyptula
 NAME = Shuffling Zombie Juror
 
 # *DOCUMENTATION*
-- 
1.7.10.4

